src/components/Learn/Learn.jsx

```diff
@@ -113,7 +113,7 @@ const Learn = () => {
           <div className="learn__important-text">
             <h3>One last thing</h3>
             <h1 className='greet practice__cp'><i>Practice, practice, practice</i></h1>
-            <span>Please watch <a href="https://www.youtube.com/watch?v=xAeiXy8-9Y8" target="_blank">How to start Competitive Programming? For beginners!</a> by Errichto.</span>
+            <span>Please watch <a href="https://www.youtube.com/watch?v=xAeiXy8-9Y8" target="_blank">How to start Competitive Programming? For beginners!</a> by Errichto.</span> 
```

Comment:
Please omit any trailing space(s)

You can either:
* Enable `editorconfig` plugin, and/or
* Run `npm run format` (not recommended for now, will format a lot of files)